db/migrate/20190710132957_rename_my_page_widgets.rb

```diff
@@ -0,0 +1,67 @@
+class RenameMyPageWidgets < ActiveRecord::Migration[5.2]
+  def up
+    Grids::MyPage.includes(:widgets, user: :preference).each do |page|
+      I18n.with_locale(page.user.language || 'en') do
```

Comment:
```suggestion
      I18n.with_locale(page.user&.language.presence || 'en') do
```

On my local community dump, users partially have an empty string as a language, or the `page.user` is nil, both times breaking the migration.

Reply:
thanks